login.php

```diff
@@ -23,10 +23,11 @@
  *
  *  fUsername
  *  fPassword
+ *  token
  *  lang
  */
 
-define('POSTFIXADMIN_LOGOUT', 1);
+//define('POSTFIXADMIN_LOGOUT', 1);
```

Comment:
If you look carefuly on the code in common.php, you will see this:

```
if (defined('POSTFIXADMIN_LOGOUT')) {
    session_unset();
    session_destroy();
    session_start();
}
```

My code stores CSRF token in `$_SESSION` and this code deletes the session (including the token).

If you actually wanted to destroy session (log out) of the old user whenever new user logs in, you would use `session_regenerate_id (true)` - the true means destroy old session.